rex/multi_year_resource.py

```diff
@@ -31,14 +31,16 @@ def __init__(self, h5_path, years=None, res_cls=Resource, hsds=False,
         h5_path : str
             Unix shell style pattern path with * wildcards to multi-file
             resource file sets. Files must have the same coordinates
-            but can have different datasets or time indexes.
+            but can have different datasets or time indexes. Can also be a path
+            on HSDS starting with /nrel/ or a path on s3 starting with s3://
         years : list, optional
             List of integer years to access, by default None
         res_cls : obj
             Resource class to use to open and access resource data
         hsds : bool
             Boolean flag to use h5pyd to handle .h5 'files' hosted on AWS
-            behind HSDS
+            behind HSDS, by default False. This is now redundant; file paths
```

Comment:
Is the idea to drop this option in the future sometime? Or should we leave it in case someone wants to use HSDS for a non `/nrel` domain? If the latter, we can probably say something to the extent of "This is redundant for the `/nrel` domain"

Reply:
Yeah i'm not 100% sure. My thinking was basically to not deprecate a kwarg right now until we've built experience with the new automatic parsing features. Trying not to make breaking changes in this PR, just new features. I would be open to dropping it in the future, maybe in an intermediate version release?